packages/terra-demographics-banner/src/_utils.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
```

Comment:
Update the _utils file name to _sharedObjects in [this commit](https://github.com/cerner/terra-core/pull/431/commits/f78d603898418000f4d27811e99d80a4951ed18c)